bin/acfg1.py

```diff
@@ -178,9 +210,11 @@ def get_or_create_ids(username, groupname):
     parser.add_argument("source", help="Source location overlay directory")
     parser.add_argument("destination", help="Destination of where to overlay source directory over")
     parser.add_argument('--dry', action='store_const', const=True, default=False, help='Dry run only')
+    parser.add_argument('--hostclass', action='append', dest='hostclasses', metavar='HOSTCLASS',
+                        help='A hostclass (possibly virtual) for which to select files. (Repeatable.)')
```

Comment:
While it was ambiguous before, might be worth telling user what happens when this is not specified.

Any reason not to pick up hostclass from file if arg is not set here?